### PR TITLE
Feature#3 create data base in firebase

### PR DESCRIPTION
**Pull request by EusebioAjas**
# Description
 
I added firebase configuration on android devices.

These changes will be completed in the future to integrate the functionalities of the issue #3 .
See #5 for more information.

 # Testing
  
  ## Screenshots 
image

### CODE REVIEW

File: src/screens/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        </View>
+      );
+    }
+  
+    return (
+      <View>
+        <Text>Welcome {user.email}</Text>
+      </View>
+    );
+  }
```
Comment (YilKanda) [inline src/screens/Login/index.js:36]: Add a blank line

File: src/library/methods/firebase.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export const createUser = () => {
+  auth()
+    .createUserWithEmailAndPassword('jane.doe@example.com', 'SuperSecretPassword!')
```
Comment (YilKanda) [inline src/library/methods/firebase.js:3]: Even if it's an example, you guys try not to upload hard data to GitHub. You can make use of parameters.

```suggestion
export const createUser = (emailUser,passwordUser) => {
  auth()
    .createUserWithEmailAndPassword(emailUser,passwordUser)
```

File: src/screens/Login/index.js (hunk 1 above)
Comment (EusebioAjas) [reply src/screens/Login/index.js]: Blank line added

**Review by YilKanda — APPROVED** [review]
LGTM!

File: src/library/methods/firebase.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    .catch(error => {
+      if (error.code === 'auth/email-already-in-use') {
+        console.log('That email address is already in use!');
+      }
+
+      if (error.code === 'auth/invalid-email') {
+        console.log('That email address is invalid!');
+      }
+
+      console.error(error);
```
[the remark this answers is not in the document]
Comment (EusebioAjas) [reply src/library/methods/firebase.js]: `console.log()` has been replaced

**Review by jkzasori — APPROVED** [review]

File: src/library/methods/firebase.js (hunk 2 above)
Comment (EusebioAjas) [reply src/library/methods/firebase.js]: Nice, parameters have been added.

**Pull request merged**